test/fake.js

```diff
@@ -2,12 +2,22 @@
  * http://creativecommons.org/publicdomain/zero/1.0/ */
 
 var crypto = require('crypto')
-var uuid = require('uuid')
 
-function hex16() { return crypto.randomBytes(16).toString('hex') }
-function hex32() { return crypto.randomBytes(32).toString('hex') }
-function hex64() { return crypto.randomBytes(64).toString('hex') }
-function hex96() { return crypto.randomBytes(96).toString('hex') }
+function hex(len) {
```

Comment:
Perhaps we can re-use the `fake` module from fxa-auth-db-server, rather than duplicating all this code into the backend.